app/controllers/records_controller.rb

```diff
@@ -1,6 +1,22 @@
 class RecordsController < ApplicationController
   include RecordsControllerBehavior
 
+  def choose_ingest_options
+    @templates = [["Raw (no template)", nil]] + FormTemplate.all.collect {|t| [t.title, t.id]}
+  end
+
+  def new
+    @form = build_form
+    find_template
+    render 'records/new'
+  end
+
+  def edit
+    @form = build_form
```

Comment:
I've got this finished and sitting, but it does mean we have `@form.template = find_template(params[:template_id])` twice (new and edit actions).  In fact, the new and edit actions are exactly the same if it weren't for that explicit `render` we need.

Ideas?  Maybe a `prepare_form` method?